lib/model/entity/profiel/Profiel.class.php

```diff
@@ -653,4 +677,10 @@ public function isInGoogleContacts() {
 		}
 	}
 
+	public function propertyMogelijk(string $name) {
+		if (!array_key_exists($name, Profiel::$properties_lidstatus)) {
```

Comment:
`static::` ?

Reply:
Met een profiel als argument dan? Of een lidstatus?

Reply:
Of bedoel je `Profiel::$properties_lidstatus`? Die is static omdat dit eigenlijk een constante is die niet afhangt van het profiel zelf. Per property bevat het informatie over voor welke lidstatussen hij van toepassing is.

Reply:
Je kan `static::$properites_lidstatus` doens, maar `Profiel::..` werkt ook prima.

Reply:
Oh :-)

Reply:
Verschil is dat ```static::``` ervoor zorgt dat runtime pas bepaald wordt welke class er gebruikt wordt en werkt dus ook met inheritance, maar maakt in dit geval niet echt uit lijkt me. Heeft vooral invloed als je static methods gaat overriden in subclasses.

Reply:
`static::` voor als je inheritance wil toestaan, `self::` als je dat niet wil en `Profiel::` als je in een andere klasse zit.

`Profiel::` en `self::` betekenen op deze plek hetzelfde.